docs/queryworkers/akamai/manage-integrations.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 10
+title: Manage Integrations
+---
+
+This page describes how to integrate Akamai's Edge Worker functionality as a Macrometa functions.
+
+:::note
+This feature is currently in preview.
+:::
+
+## Prerequisites
+
+- A Macrometa account with sufficient permissions to view and edit query workers.
+- In Akamai, a Property created with your Macrometa GDN integration. For more information about Akamai Properties, refer to [Create and configure a property](https://techdocs.akamai.com/api-definitions/docs/create-config-prop).
+
+## Add Integration
```

Comment:
After the new API is rolled out, let's see if there's an API way to do this and add a tab. Something for next week.